asserts/repair.go

```diff
@@ -42,10 +42,10 @@ func (r *Repair) BrandID() string {
 	return r.HeaderString("brand-id")
 }
 
-// RepairID returns the "id" of the repair. It should be a short string
-// that follows a convention like "REPAIR-123". Similar to a CVE there
-// should be a public place to look up details about the repair-id
-// (e.g. the snapcraft forum).
+// RepairID returns the "id" of the repair. It should be a short
+// string ID like "123". Similar to a CVE there should be a public
```

Comment:
Shouldn't this be an int? It was originally a string when we had REPAIR-123, but then we decided to have those as two separate things, becoming brand REPAIR with id N, and there's necessarily an order for the second one since that's how we pick new repairs. A string would order incorrectly.

Probably for a follow up, though.

Reply:
Yeah, it sounds like a follwup to me.

Reply:
Yes, in that branch we can also add summary to repairs